### PR TITLE
[master]Support MTU configuration for VM's NIC in RHEL

### PR DESCRIPTION
**Pull request by zhshuj**
Add code logic to parse MTU value from network info
and set to network configuration scripts for RHEL.

Signed-off-by: Shu Juan Zhang <zshujuan@cn.ibm.com>

### CODE REVIEW

**Comment by zhshuj** [comment]
@SeanHQF @jichenjc @bjhuangr  Please help review.

File: zvmsdk/dist.py [hunk 1]
```diff
@@ -121,8 +121,9 @@ def create_network_configuration_files(self, file_path, guest_networks,
     def _generate_network_configuration(self, network, vdev, active=False):
         ip_v4 = dns_str = gateway_v4 = ''
         ip_cidr = netmask_v4 = broadcast_v4 = ''
-        net_cmd = ''
+        net_cmd = mtu = ''
```
Comment (jichenjc) [inline zvmsdk/dist.py:124]: we inited the mtu here

so if there is no MTU input, what's the output at line 351?

Comment (zhshuj) [reply zvmsdk/dist.py:124]: The output of line 351 is:
MTU=""

And write this line to /etc/sysconfig/network-scripts/ifcfg-<nic>
And deployed VM has MTU value as 1500

Comment (jichenjc) [reply zvmsdk/dist.py:124]: ok, default is 1500 so "" means auto select.. thanks for the confirmation

**Pull request merged**